cross/libvpx/Makefile

```diff
@@ -28,40 +30,33 @@ include ../../mk/spksrc.archs.mk
 
 ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
-ADDITIONAL_CPPFLAGS = -std=c++0x
 ADDITIONAL_CXXFLAGS = -std=c++0x
 endif
 
 ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=armv7-linux-gcc
-CONFIGURE_ARGS += --disable-neon
 endif
 
 ifeq ($(findstring $(ARCH),$(PPC_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
 endif
 
-# neon_asm does not work under armv8/arm64
 ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=armv8-linux-gcc
 endif
 
-# Define x86asm
-ifeq ($(findstring $(ARCH),$(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
-BUILD_DEPENDS = native/nasm
-NASM_PATH = $(abspath $(PWD)/../../native/nasm/work-native/install/usr/local/bin)
-ENV += AS=$(NASM_PATH)/nasm
-ENV += PATH=$(NASM_PATH):$$PATH
-CONFIGURE_ARGS += --as=nasm
-CONFIGURE_ARGS += --disable-sse4_1
-endif
-
 ifeq ($(findstring $(ARCH),$(i686_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=x86-linux-gcc
+# --as=auto: prefers yasm over nasm
+CONFIGURE_ARGS += --as=auto
+CONFIGURE_ARGS += --disable-sse4_1
```

Comment:
@th0ma7 can you remember why you disabled sse4_1 for intel archs?
_it builds just fine without disabling it_

Reply:
This change comes from this PR: https://github.com/SynoCommunity/spksrc/pull/6148
Specifically from this commit: https://github.com/SynoCommunity/spksrc/pull/6148/commits/c4c0d44f0066f7ff0fffb1516e35f7d2f9032260

But I haven't left any descriptions as to why I disabled it?  Maybe it failed on DSM6, again using gcc <= 4.9.x for that specific libvpx version (i.e 1.14.1)?

Looking a bit further this may have been a mistake from my part as SSE4.1 (Streaming SIMD Extensions 4.1) has nothing to do with newer AVX extensions and may have mixed them somehow.  A bit more reading on it makes it that pretty much all "recent" intel based nas from synology (i.e. less than 10-12 years old) should support this extension...

I believe it is safe to remove that.